frontend/src/components/body.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import doodleImage from "../assets/doodle.png";
+import logo from "../assets/logo.png";
+import "./body.css";
+const Body: React.FC = () => {
+  return (
+    <>
+      <div className="main">
+        <div
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            alignItems: "center",
+          }}
```

Comment:
add this inside .css file
btw, multiple css files not needed in this case, just import one css file.